subscriptions.go

```diff
@@ -89,6 +91,36 @@ type SubscriptionAPI struct {
 	backOffConf backOffConfiguration
 }
 
+type PageParameters struct {
+	Limit int
+	Offset int
+}
+
+type SubscriptionParameters struct {
+	PageParameters
+	OwningApplication string
+	EventType         string
+}
+
+func (p *SubscriptionParameters) EncodeQueryString() string {
+	params := url.Values{}
+
+	if len(p.OwningApplication) > 0 {
```

Comment:
Comparing for empty string is a bit clearer and more idiomatic. Also applies to check of `p.EventType` below.